docs/useGetRecordId.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: "useGetRecordId"
+---
+
+# `useGetRecordId`
+
+This hook returns the current `recordId`. `recordId` is obtained from parameters if passed as a parameter, or from the `RecordContext` if there is one, or, lastly, from the react-router URL.
+
+`useGetRecordId` takes an optional `recordId` argument if used inside a `RecordContextProvider` or if `recordId` can be guessed from the URL
```

Comment:
I'm not fond of the wording, which suggests the `recordId` argument is needed if used inside a `RecordContextProvider` or if `recordId` can be guessed from the URL, when it is actually the opposite.